internal/deb/extract.go

```diff
@@ -185,7 +186,7 @@ func extractData(dataReader io.Reader, options *ExtractOptions) error {
 				// Base directory for extracted content. Relevant mainly to preserve
 				// the metadata, since the extracted content itself will also create
 				// any missing directories unaccounted for in the options.
-				err := fsutil.Create(&fsutil.CreateOptions{
+				err := options.FSCreator.Create(&fsutil.CreateOptions{
```

Comment:
Is there a place validating this so it doesn't panic if nil, maybe setting it to a default value?

Reply:
I didn't add it because it was an internal function, I have add it now.